packages/@aws-cdk/aws-events-targets/lib/batch.ts

```diff
@@ -0,0 +1,81 @@
+import * as events from '@aws-cdk/aws-events';
+import * as batch from '@aws-cdk/aws-batch';
+import * as iam from '@aws-cdk/aws-iam';
+import { singletonEventRole } from './util';
+
+/**
+ * Customize the Batch Job Event Target
+ */
+export interface BatchJobProps {
+  /**
+   * The event to send to the Lambda
+   *
+   * This will be the payload sent to the Lambda Function.
+   *
+   * @default the entire CloudWatch event
+   */
+  readonly event?: events.RuleTargetInput;
+
+  /**
+   * The size of the array, if this is an array batch job.
+   * 
+   * Valid values are integers between 2 and 10,000.
+   * 
+   * @default undefined
+   */
+  readonly size?: number;
+
+  /**
+   * The number of times to attempt to retry, if the job fails. Valid values are 1–10. 
+   * 
+   * @default undefined
+   */
+  readonly attempts?: number;
+}
+
+/**
+ * Use an AWS Batch Job / Queue as an event rule target.
+ */
+export class BatchJob implements events.IRuleTarget {
+  constructor(
+    public readonly jobQueue: batch.IJobQueue,
+    public readonly jobDefinition: batch.IJobDefinition,
+    private readonly props: BatchJobProps = {}
+  ) { }
+  
+  /**
+   * Returns a RuleTarget that can be used to trigger queue this batch job as a
+   * result from a CloudWatch event.
+   */
+  public bind(rule: events.IRule, _id?: string): events.RuleTargetConfig {
+    let batchParameters: events.CfnRule.BatchParametersProperty = {
+      jobDefinition: this.jobDefinition.jobDefinitionArn
+    }
+
+    if (this.props.size) {
+      batchParameters['arrayProperties'] = {
+        size: this.props.size
+      }
+    }
+
+    if (this.props.attempts) {
+      batchParameters['retryStrategy'] = {
```

Comment:
`batchParameters.retryStrategy` ?

Reply:
See here: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-batchparameters.html

RetryStrategy is an optional object